cinema/serializers.py

```diff
@@ -22,3 +27,48 @@
         instance.save()
 
         return instance
+
+
+class ActorSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    first_name = serializers.CharField(max_length=100)
+    last_name = serializers.CharField(max_length=100)
+
+    def create(self, validated_data):
+        return Actor.objects.create(**validated_data)
+
+    def update(self, instance, validated_data):
+        instance.first_name = validated_data.get("first_name", instance.first_name)
+        instance.last_name = validated_data.get("last_name", instance.last_name)
+        instance.save()
+        return instance
+
+
+class GenreSerializer(serializers.Serializer):
```

Comment:
Consider using `serializers.ModelSerializer` for the `GenreSerializer` class to reduce the need for manually defining the `create` and `update` methods.